skops/io/_scikeras.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import io
+import os
+import tempfile
+from typing import Sequence, Type
+
+import tensorflow as tf
+from scikeras.wrappers import KerasClassifier, KerasRegressor
+
+from ._audit import Node
+from ._protocol import PROTOCOL
+from ._utils import Any, LoadContext, SaveContext, get_module
+
+
+def scikeras_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
+    res = {
+        "__class__": obj.__class__.__name__,
+        "__module__": get_module(type(obj)),
+        "__loader__": "SciKerasNode",
+    }
+
+    obj_id = save_context.memoize(obj)
+    f_name = f"{obj_id}.keras"
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        file_name = os.path.join(temp_dir, "model.keras")
+        obj.model.save(file_name)
```

Comment:
so we only save the `model` attribute? This sounds odd.

Reply:
From my understanding of https://keras.io/guides/serialization_and_saving/, it seems that Keras is compressing all the pieces of the models into the `.keras` file. Should I change the name of the file to make it more clear?